homework_1/zadatak1/fizzbuzz/fizzbuzz.go

```diff
@@ -0,0 +1,43 @@
+package fizzbuzz
+
+import (
+	"github.com/pkg/errors"
+	"strconv"
+)
```

Comment:
Usually we want to separate imports by their type into groups with newlines between them. The three groups we choose usually are:
* standard library imports (in this case `strconv`)
* third party libraries (in this case `github.com/pkg/errors`)
* this repository

This really comes in handy when there are files with tens of imports. It can get cluttered really quickly otherwise.
So I would suggest changing this to 
```suggestion
import (
       "strconv"
       
	"github.com/pkg/errors"
)
```